### PR TITLE
Aiyan's Develop

### PR DESCRIPTION
**Pull request by krishnadurai**

### CODE REVIEW

File: cache.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package cache
+package aurora_test_auth
```
Comment (krishnadurai) [inline cache.go:1]: We should retain the package's name to `cache`

File: config.template.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+cache:
```
Comment (krishnadurai) [inline config.template.yaml:1]: 12-factor application favours ENV over configs.

File: internal/cache/config.go [hunk 3]
```diff
@@ -1,24 +0,0 @@
-package cache
```
Comment (krishnadurai) [inline internal/cache/config.go:1]: Please retain this.

File: internal/cache/redis/redis.go [hunk 4]
```diff
@@ -0,0 +1,74 @@
+package redis
+
+import (
+	"context"
+	"github.com/krishnadurai/aurora-test-auth"
```
Comment (krishnadurai) [inline internal/cache/redis/redis.go:5]: Wrong order of imports. Please run `go fmt`.

**Review by krishnadurai — COMMENTED** [review]
@aiyan we need to follow https://12factor.net/

File: config.template.yaml (hunk 2 above)
Comment (aiyan) [reply config.template.yaml:1]: Yep we can change this

File: cache.go (hunk 1 above)
Comment (aiyan) [reply cache.go:1]: Sure

File: internal/cache/config.go (hunk 3 above)
Comment (aiyan) [reply internal/cache/config.go:1]: Sure

File: internal/cache/redis/redis.go (hunk 4 above)
Comment (aiyan) [reply internal/cache/redis/redis.go:5]: Sure

**Comment by aiyan** [comment]
Krishna and I decided to use the master branch and environment variables. Closing this as we'll be making changes off master

**Pull request closed without merging**